extensions/wikia/AdEngine/js/utils/env.js

```diff
@@ -0,0 +1,13 @@
+define('ext.wikia.adEngine.utils.env', [
+	'wikia.window'
+], function (win) {
+	'use strict';
+
+	function isDevEnvironment() {
+		return /.+wikia-dev\.com$/.test(win.location.host);
+	}
+
+	return {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)